File: c++/0020-valid-parentheses/main.cpp
#include <cstdlib>
#include <iostream>
#include <string>

#include "solution.cpp"

using namespace std;

int main() {
    Solution solution;

    string solution1 = solution.isValid("()") ? "true" : "false";
    cout << "true: " << solution1 << endl;

    string solution2 = solution.isValid("()[]{}") ? "true" : "false";
    cout << "true: " << solution2 << endl;

    string solution3 = solution.isValid("(]") ? "true" : "false";
    cout << "false: " << solution3 << endl;

    return EXIT_SUCCESS;
}

File: java/0020-valid-parentheses/Main.java
public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        System.out.println("true: " + solution.isValid("()"));
        System.out.println("true: " + solution.isValid("()[]{}"));
        System.out.println("false: " + solution.isValid("(]"));
    }
}
